RELEASE_NOTES.md

```diff
@@ -18,6 +18,12 @@
 - The following gRPC method has been renamed:
   `StreamComponentData` -> `SubscribeComponentData`
 
+- Added support for a new Component category: Relays. These are electromagnetic
+  switches that control circuit breakers in the microgrid, e.g., to connect or
+  disconnect an inverter from the grid. Relays support the following methods:
```

Comment:
Relays are not just for disconnecting inverters? We could also use relays for other stuff.Is it on purpose to only focus on inverters here?